SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -187,6 +209,14 @@ void ASpatialFunctionalTest::StartTest()
 
 void ASpatialFunctionalTest::FinishStep()
 {
+	// We can only FinishStep if there are no SoftAssert fails.
```

Comment:
Maybe worth putting this in a comment on the FinishStep() header

Reply:
Yes, I'll improve that comment.